apps/OboeTester/app/src/main/java/com/mobileer/oboetester/RoundTripLatencyActivity.java

```diff
@@ -195,15 +216,28 @@ public String getLastReport() {
         private Runnable runnableCode = new Runnable() {
             @Override
             public void run() {
-                String message;
+                double timestampLatency = -1.0;
+                int state = getAnalyzerState();
+                if (state == STATE_MEASURE_BACKGROUND || state == STATE_IN_PULSE) {
+                    timestampLatency = measureTimestampLatency();
+                    if (timestampLatency > 0) {
```

Comment:
Would it be useful to keep track of the latency count somewhere in case this keeps on returning 0 or less?

Reply:
Some configurations do not support input timestamps.
Negative latency is impossible so I use -1 to indicate an error or no measurement available.